.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
I don't think this file should be added?!

Reply:
Indeed. @geekpulp You should add `.vscode` to your global `.gitignore` file :-)